Dodo SketchBoard

codingkatty opened the pull request:
# BrowserBuddy Extension Submission

## Project Details

1. **What is the name of your project?**
Dodo SketchBoard

2. **How many hours did you spend developing this?**  
7 hours

3. **What problem does your extension solve, or what makes it unique?**  
It is a floating window that could be used to jot down notes and use as reference. The window can be opened in any tab, and the content is stored and synced, making it easy to use. Example of using it is reference for drawing, drafting, and its basically a flexible notepad in your browser.

4. **How many people have tested your extension?**  
2 (last version, I then added stuff according to feedback)

5. **Does your extension work in both Chrome and Firefox?**  
Yes

---

## Additional Comments

(Feel free to provide any additional context or notes here)

JosiasAurel reviewed (changes requested):
So it's difficult to read the text in the sketchpad when on a page with white background. 
<img width="520" alt="image" src="https://github.com/user-attachments/assets/efa05725-95e1-49e9-8e0d-10f74615232a" />

I also think it'll be nice to change the font to something a bit more readable like Inter or allow the user to chose their own font among a small selection. 

The last feedback I have is that it'll be nice if one could create different notepads instead of it being the same notepad you're writing into across all websites.

codingkatty commented:
Hi, thanks for the review! Actually the color can be changed with the toggle, sorry if it was not really clear. I'll try to improve it

codingkatty commented:
Hi, so I made a few changes, but I'm not really sure how to make the multiple notepads because then it would be hard to save/open (maybe I could do it in an update). Instead, I've added a clear button so its easy to get a clean new window.

Here's a list of functions so far: dark/light mode toggle (I've added a symbol), save (chrome.storage), resize, drag and drop images & change fonts
![image](https://github.com/user-attachments/assets/704c7243-1048-499f-830f-04c1a906edda)

JosiasAurel reviewed (approved):
Nice! Please fill [this form](https://airtable.com/appgtKZwZBoOeFU7O/pagyeS51zr2kC1rp0/form) to get your grant.
